rust/kcl-lib/src/parsing/ast/types/mod.rs

```diff
@@ -125,7 +125,7 @@ impl<T> Node<T> {
         }
     }
 
-    pub fn boxed(inner: T, start: usize, end: usize, module_id: ModuleId) -> BoxNode<T> {
```

Comment:
What was the purpose of this change?

Reply:
I changed a call from new to boxed and not changing the order meant factoring out start/end/id variables, so I thought it was better that the two ctor sigs matched